web/src/components/filters/filters-toolbar.tsx

```diff
@@ -260,35 +146,51 @@ export const FiltersToolbar: React.FC<FiltersToolbarProps> = ({
+        {!isForced && quickFilters.length > 0 && (
           <ToolbarItem className="flex-start">
-            <QuickFilters quickFilters={quickFilters} activeFilters={filters || []} setFilters={setFilters} />
+            <QuickFilters quickFilters={quickFilters} activeFilters={filters?.list || []} setFilters={setFiltersList} />
           </ToolbarItem>
         )}
-        {_.isEmpty(forcedFilters) && (
-          <ToolbarItem className="flex-start">
-            <Tooltip
-              //css hide tooltip here to avoid render issue
-              className={`filters-tooltip${_.isEmpty(message) ? '-empty' : ''}`}
-              isVisible={!_.isEmpty(message)}
-              content={message}
-              trigger={_.isEmpty(message) ? 'manual' : 'click'}
-              enableFlip={false}
-              position={'top'}
-            >
-              <div>
-                <InputGroup>
-                  <FiltersDropdown
-                    selectedFilter={selectedFilter}
-                    setSelectedFilter={setSelectedFilter}
-                    allowConnectionFilter={allowConnectionFilter}
+        {!isForced && (
+          <>
+            <ToolbarItem className="flex-start">
+              <Tooltip
+                //css hide tooltip here to avoid render issue
+                className={`filters-tooltip${_.isEmpty(message) ? '-empty' : ''}`}
+                isVisible={!_.isEmpty(message)}
+                content={message}
+                trigger={_.isEmpty(message) ? 'manual' : 'click'}
+                enableFlip={false}
+                position={'top'}
+              >
+                <div>
+                  <InputGroup>
+                    <FiltersDropdown
+                      selectedFilter={selectedFilter}
+                      setSelectedFilter={setSelectedFilter}
+                      allowConnectionFilter={allowConnectionFilter}
+                    />
+                    {getFilterControl()}
+                  </InputGroup>
+                  <FilterHints def={selectedFilter} />
+                </div>
+              </Tooltip>
+            </ToolbarItem>
+            {filters && (
+              <ToolbarItem className="flex-start" id="back-and-forth-item">
+                <Tooltip content={t('Include return traffic, with swapped source and destination filters')}>
+                  <Checkbox
+                    data-test="back-and-forth"
+                    id="back-and-forth"
+                    onChange={setBackAndForth}
+                    isChecked={filters.backAndForth}
+                    label={t('Back and forth')}
+                    aria-label={t('Back and forth')}
                   />
-                  {getFilterControl()}
-                </InputGroup>
-                <FilterHints def={selectedFilter} />
-              </div>
-            </Tooltip>
-          </ToolbarItem>
+                </Tooltip>
+              </ToolbarItem>
```

Comment:
Same here maybe it's the time to allow `back and forth` and `show / hide filters` in the "more options" dropdown ?

![image](https://github.com/netobserv/network-observability-console-plugin/assets/91894519/f96b13d3-7e85-423b-9bf3-d496139d7e7c)

Reply:
or maybe place the `back and forth` check somewhere else ?
- in the query options (but would be less visible)
- at swap filters level (would be more consistent to me since it's related to existing filters)